docs/pages/concepts/backend-lifecycle.mdx

```diff
@@ -0,0 +1,37 @@
+# Backend lifecycle
+
+The backend lifecycle refers to the sequence of statuses that a backend managed by Plane passes through,
+and defines Plane’s responsibilities at each stage.
+
+The statuses are:
+
+- `Scheduled`: The Plane controller has chosen a drone to run the backend.
+- `Loading`: The drone has acknowledged the backend and is loading the appropriate image from the registry.
+- `Starting`: The drone has loaded the image and is starting the container.
+- `Waiting`: The container has started. The drone is waiting for it to listen on an HTTP port.
+- `Ready`: The container is listening on an HTTP port. The drone is ready to route traffic to it.
+- `Terminating`: The drone has sent a request to terminate the backend. If the request was a “soft” request,
+  the backend may remain in this state for a grace period (by default 10 seoconds) before being hard-terminated.
+- `Terminated`: The drone has terminated the backend. This is considered the only terminal state.
```

Comment:
What about `Swept`, `Exited`, `Failed`, and other error states?

Reply:
In Plane2 these are all wrapped into `Terminated`, but other data is stored (like exit code) that can be used to determine the nature of an exit. I am open to exposing that through the API, although we don't yet.